### ISSUE TITLE
Game Science Update Breaks Ultrawide Fix, Causes Main Menu Crashes

### ISSUE THREAD

**Issue by VintageGenetics** [Lyall/WukongTweak#15]
Game Science has recently released an update for the game v**1.0.8.14860,** which has caused the ultrawide fix to stop functioning. As a result, the game now crashes to the desktop when attempting to access the main menu.

**Comment by Brenovyski** [comment]
Same issue here. After the update, I had to remove the fix to be able to play it.

**Comment by Lyall** [comment]
The crash is related to enabling the developer console. To fix it for now, disable the option in `WukongTweak.ini`

```ini 
[Developer Console]
; Enable developer console. 
; If the console does not open, check the README or "WukongTweak.log" for info on how to enable it.
Enabled = false
```

**Comment by Lyall** [comment]
Fixed in v0.8.4.
